### PR TITLE
Adding mysql configuration facts module

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Adding MySQL configuration facts module
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME
azure_rm_mysqlconfiguration_facts

##### ANSIBLE VERSION
2.7

### CODE REVIEW

**Comment by Fred-sun** [comment]
kindly ping

File: lib/ansible/modules/cloud/azure/azure_rm_mysqlconfiguration_facts.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+        self.mgmt_client = None
+        self.resource_group = None
+        self.server_name = None
+        self.name = None
+        super(AzureRMMySQLConfigurationFacts, self).__init__(self.module_arg_spec, supports_tags=False)
+
+    def exec_module(self, **kwargs):
+        for key in self.module_arg_spec:
+            setattr(self, key, kwargs[key])
+        self.mgmt_client = self.get_mgmt_svc_client(MySQLManagementClient,
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_mysqlconfiguration_facts.py:139]: api version

**Review by yungezz — COMMENTED** [review]
looks good. one minor comment.

File: lib/ansible/modules/cloud/azure/azure_rm_mysqlconfiguration_facts.py (hunk 1 above)
Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_mysqlconfiguration_facts.py:139]: there's no api_version parameter for MySQL client, there's only one, 2017-12-01

https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-rdbms/azure/mgmt/rdbms/mysql/my_sql_management_client.py

Comment (yungezz) [reply lib/ansible/modules/cloud/azure/azure_rm_mysqlconfiguration_facts.py:139]: what does that mean? use default for now, and upgrade in future if there's new version?

**Review by yungezz — APPROVED** [review]

**Pull request merged**